infra/terraform/k8s-local/run.sh

```
@@ -1,58 +1,111 @@
 set -euo pipefail
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 PROJECT_ROOT="$(cd "$SCRIPT_DIR/../../.." && pwd)"
 FE_ROOT="$(cd "$PROJECT_ROOT/../stablebridge-web" 2>/dev/null && pwd || echo "")"
-INFRA_TF_DIR="$SCRIPT_DIR/../local"
-K8S_TF_DIR="$SCRIPT_DIR"
-
-CLUSTER_NAME="stablebridge-local"
 
-# All images to load into kind
 IMAGES=(
```

Shellcheck (0.11.0)

[warning] IMAGES appears unused. Verify use (or export if used externally).

(SC2034)